webbpsf/roman.py

```diff
@@ -871,7 +871,7 @@ def unlock_pupil_mask(self):
         self._update_pupil() # reset pupil mask
 
 
-class CGI(RomanInstrument):
+class Coronagraph(RomanInstrument):
```

Comment:
My one hesitancy is just calling this class `Coronagraph` is maybe potentially confusing or ambiguous, as there are also coronagraphs on JWST. I'd suggest going with `RomanCoronagraph` to be explicit and unambiguous, even though it's a big longer.

Reply:
That makes sense on further thought. I followed through with `RomanCoronagraph` in a new commit.